yt_dlp/extractor/vrt.py

```diff
@@ -88,25 +100,22 @@ def _extract_formats_and_subtitles(self, data, video_id):
         return formats, subtitles
 
     def _call_api(self, video_id, client='null', id_token=None, version='v2'):
-        player_info = {'exp': (round(time.time(), 3) + 900), **self._PLAYER_INFO}
-        player_token = self._download_json(
-            'https://media-services-public.vrt.be/vualto-video-aggregator-web/rest/external/v2/tokens',
-            video_id, 'Downloading player token', headers={
-                **self.geo_verification_headers(),
-                'Content-Type': 'application/json',
-            }, data=json.dumps({
-                'identityToken': id_token or {},
-                'playerInfo': jwt_encode_hs256(player_info, self._JWT_SIGNING_KEY, headers={
-                    'kid': self._JWT_KEY_ID
-                }).decode()
-            }, separators=(',', ':')).encode())['vrtPlayerToken']
+        # player_info = {'exp': (round(time.time(), 3) + 900), **self._PLAYER_INFO}
+        # player_info_jwt = jwt_encode_hs256(player_info, self._JWT_SIGNING_KEY, headers={
+        #   'kid': self._JWT_KEY_ID
+        #   }).decode()
+
+        json_response = self._download_json(
+            f'https://media-services-public.vrt.be/vualto-video-aggregator-web/rest/external/{version}/tokens',
+            None, 'Downloading player token', 'Failed to download player token', headers={'Content-Type': 'application/json'}, data=json.dumps({'identityToken': id_token or self._get_cookies("https://www.vrt.be").get("vrtnu-site_profile_vt").value}).encode())
+        player_token = json_response['vrtPlayerToken']
```

Comment:
Line too long and use single quotes.
If only a single token is required, I would move this to `_real_initialize`.